### ISSUE TITLE
Class .modal-open doesn't appear when open modal from another modal.

### ISSUE THREAD

**Issue by jek-fdrv** [twbs/bootstrap#18570]
Problem in `.modal-open` class which doesn't appear after clicking on 
```html
data-dismiss="modal" data-toggle="modal" data-target="#modal-2"
```
Please check this https://jsbin.com/xonuqamuli/edit?html,output

**Comment by twbs-lmvtfy** [comment]
Hi @jek-fdrv!

You appear to have posted a live example (https://jsbin.com/xonuqamuli/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 59, column 32 thru column 36: End tag ```` br ````.
* line 14, column 3 thru column 110: The ```` aria-labelledby ```` attribute must point to an element in the same document.
* line 32, column 3 thru column 110: The ```` aria-labelledby ```` attribute must point to an element in the same document.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #17956.
